src/gatling/simulations/scenarios/CourtAdmin_PRL_FL401.scala

```diff
@@ -0,0 +1,976 @@
+        .get("/api/prd/lov/getLovRefData?categoryId=HearingType&serviceId=ABA5&isChildRequired=N")
+        .headers(Headers.commonHeader)
+        .header("Accept", "application/json, text/plain, */*")
+        .check(substring("HearingType")))
+
+      .exec(Common.caseActivityPost)
+    }
+
+    .pause(MinThinkTime, MaxThinkTime)
+
+/*======================================================================================
+* Select Request a Hearing
+=======================================================================================*/
+
+    .group("XUI_PRL_FL401Progress_320_RequestHearing") {
+
+      exec(Common.caseActivityPost)
+      .exec(Common.isAuthenticated)
+
+      .exec(http("XUI_PRL_FL401Progress_320_005_GetCaseFlag")
+        .get("/api/prd/caseFlag/getCaseFlagRefData?serviceId=ABA5")
+        .headers(Headers.commonHeader)
+        .header("Accept", "application/json, text/plain, */*")
+        .check(status.in(200, 304)))
+
+      .exec(http("XUI_PRL_FL401Progress_320_010_LocationById")
+        .get("/api/prd/location/getLocationById?epimms_id=#{locationId}&serviceCode=null") //*CORELE
+        .headers(Headers.commonHeader)
+        .header("Accept", "application/json, text/plain, */*")
+        .check(status.in(200, 304)))
+  
+    }
+
+    .pause(MinThinkTime, MaxThinkTime)
+
+/*======================================================================================
+* Navigate through hearing screens and submit hearing request
+=======================================================================================*/
+
+      .exec(http("XUI_PRL_FL401Progress_330_GetCaseFlag")
+        .get("/api/prd/caseFlag/getCaseFlagRefData?serviceId=ABA5")
+        .headers(Headers.commonHeader)
+        .header("Accept", "application/json, text/plain, */*")
+        .check(status.in(200, 304)))
+
+
+      .pause(MinThinkTime, MaxThinkTime)
+
+      .exec(http("XUI_PRL_FL401Progress_340_GetHearingChannel")
+        .get("/api/prd/lov/getLovRefData?categoryId=HearingChannel&serviceId=ABA5&isChildRequired=N")
+        .headers(Headers.commonHeader)
+        .header("Accept", "application/json, text/plain, */*")
+        .check(substring("HearingChannel")))
+
+      .pause(MinThinkTime, MaxThinkTime)
+
+      .exec(http("XUI_PRL_FL401Progress_350_LocationByIdServiceCode")
+        .get("/api/prd/location/getLocationById?epimms_id=#{locationId}&serviceCode=ABA5") 
+        .headers(Headers.commonHeader)
+        .header("Accept", "application/json, text/plain, */*")
+        .check(status.in(200, 304)))
+
+      .pause(MinThinkTime, MaxThinkTime)
+
+      .exec(http("XUI_PRL_FL401Progress_360_GetHearingChannel")
+        .get("/api/prd/lov/getLovRefData?categoryId=JudgeType&serviceId=ABA5&isChildRequired=N")
+        .headers(Headers.commonHeader)
+        .header("Accept", "application/json, text/plain, */*")
+        .check(substring("JudgeType")))
+
+      .pause(MinThinkTime, MaxThinkTime)
```

Comment:
Again, I don't think these pauses are needed as these API calls aren't invoked by the user?